### PR TITLE
use api user login

### PR DESCRIPTION
**Pull request by ghost**
## Description
User login
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
 NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

### CODE REVIEW

File: public/js/user/userLogin.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+            success: function (data){
+                window.localStorage.setItem('access_token', data.data.token);
+            },
+            error: function (data) {
+                alert(data.responseJSON.error);
+            }
+        });
+    });
+
+});
```
Comment (at-dungle) [inline public/js/user/userLogin.js]: xuống dòng

**Comment by at-dungle** [comment]
LGTM

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by at-dungle** [comment]
LGTM

**Comment by at-vietpham** [comment]
LGTM 🍦

**Pull request merged**